assets/js/ajax_search_select.js

```diff
@@ -1,85 +1,100 @@
 $(function() {
```

Comment:
I realize this file's indentation violates the style guide (4 spaces instead of 2), but I thought it would be better to fix that after the PR has been reviewed (but before merging), because otherwise it would make the diff really messy and hard to read.